src/proto/feature_flag.proto

```diff
@@ -3,6 +3,18 @@ option optimize_for = LITE_RUNTIME;
 
 package otbr;
 
+enum otbrProtoLogLevel
```

Comment:
```suggestion
enum LogLevel
```
- Use CamelCase.
- I think it may not be necessary to emphasize `otbr` and `proto` in the name.

Reply:
Agree 'otbr' is not necessary, I kept 'proto' to indicate this is for the proto file.